Python/ceval.c

```diff
@@ -880,7 +894,7 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, _PyInterpreterFrame *frame, int
     stack_pointer = _PyFrame_GetStackPointer(frame);
 
 #ifdef LLTRACE
-    lltrace = maybe_lltrace_resume_frame(frame, &entry_frame, GLOBALS());
+    lltrace = maybe_lltrace_resume_frame(frame, GLOBALS());
```

Comment:
Since we are setting `lltrace` every time we resume a frame, we could add a bit field to the frame (in debug builds only) for `lltrace`.